integration-cli/docker_cli_service_logs_test.go

```diff
@@ -360,7 +358,7 @@ func (s *DockerSwarmSuite) TestServiceLogsDetails(c *testing.T) {
 		// busybox image, shell string
 		"busybox", "sh", "-c",
 		// make a log line
-		"echo LogLine; while true; do sleep 1; done;",
+		"trap 'exit 0' TERM; echo LogLine; while true; do sleep 1; done;",
```

Comment:
we can't set `--init` for services (using the 17.06 cli), but alternatively, you could start the swarm daemon in this test with `--init` (see https://github.com/moby/moby/pull/38571)

Reply:
Thought about it, but this is really introducing another variable to the tests. I like the `trap` here.